src/SequentialBackend.jl

```diff
@@ -198,3 +198,7 @@ function async_exchange!(
   end
   t_out
 end
+
+# Do not generate warning if SequentialBackend
+function _warn_message_on_main_task_discover_parts_snd(data::SequentialData)
```

Comment:
I think we also want the warning in the sequential back-end. We want to discover this warning as soon as possible. Better when we still are in sequantial mode

Reply:
Ok. Agreed. Makes sense. Done.